index.html

```diff
@@ -114,6 +114,79 @@ <h4 class="heading heading_level-4">Подушка</h4>
                 </div>
             </div>
         </section>
+        
+        <section class="features">
+            <div class="column-wrapper">
+                <div class="column-row">
+                    <div class="features__list">
+                        <div class="column column_s-2 column_m-2 column_l-4">
+                            <div class="features__item">
+                                <img class="features__icon" src="./images/delivery-icon.svg" alt="Быстрая доставка"/>
+                                <p class="features__name">Быстрая доставка по всей России</p>
+                            </div>
+                        </div>
+                        <div class="column column_s-2 column_m-2 column_l-4">
+                            <div class="features__item">
+                                <img class="features__icon" src="./images/return-icon.svg" alt="100% гарантия возврата"/>
+                                <p class="features__name">100% гарантия возврата и подлинность товара</p>
+                            </div>
+                        </div>
+                        <div class="column column_s-2 column_m-2 column_l-4">
+                            <div class="features__item">
+                                <img class="features__icon" src="./images/support-icon.svg" alt="Круглосуточная служба поддержки"/>
+                                <p class="features__name">Круглосуточная служба поддержки</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <footer class="footer">
+            <div class="column-wrapper">
+                <div class="column-row">
+                    <div class ="footer__main">
+                        <div class="column column_s-2 column_m-3 column_l-4">
+                            <nav class="footer__menu">
```

Comment:
класс  ```footer__menu``` удалил из стилей, а отсюда забыл
```suggestion
                            <nav>
```